docs/UserGuide.md

```diff
@@ -1082,9 +1085,12 @@ Fig 11. Revenue bar chart
 Fig 12. Client pie chart
 {:.caption}
 
-#### 4.7.3 Viewing top 10 recipes
-* Your top 10 recipes that appear in the highest number of orders will be down in a pie chart.
-* Ties are broken arbitrarily i.e. If there are multiple recipes with the same number of orders, 10 random recipes
+#### 4.7.3 Viewing top 10 most popular recipes
+* Your top 10 most popular recipes, that appear most frequently in the all your orders will be shown in a pie chart.
+* To rank the popularity of orders, compare the total order quantity of each recipe. <br>
+  (e.g. Compare an order list with only these 3 orders: 'Chicken Rice x4', 'Fried Rice x2' and 'Fried Rice x1' <br>
+  Chicken rice is more popular since its total quantity (sum of all `ORDER_QUANTITY`) is higher, i.e. 4 > 2+1.)
```

Comment:
```suggestion
  Chicken rice is more popular since its total quantity (sum of all its `ORDER_QUANTITY`) is higher, i.e. 4 > 2 + 1.)
```

dk if this better. up to you i think.